app/templates/admin/createSingleEvent.html

```diff
@@ -116,17 +118,17 @@ <h1 id="title">{{page_title}}</h1>
           <div class="form-inline d-flex">
             <div  class="form-group" >
               <div class="d-inline-flex">
-                <label for="startTime"><strong>Start Time:</strong></label>
+                <label for="startTimePicker"><strong>Start Time:</strong></label>
                 {% if isEditable %}
-                  <input type='time' class="form-control timePicker" id='pickStartTime'value="{{eventData.timeStart}}"  name="timeStart" required>
+                  <input type='time' class="form-control" id='startTime'value="{{eventData.timeStart}}"  name="timeStart" onchange=updateTime(this) required>
```

Comment:
The label doesn't match the id anymore

Reply:
Also, since we're already doing stuff to these elements in javascript, including adding an event listener, the onchange event should also be added there.

Reply:
made label match the id. removed onchange event